internal/support/logging/init.go

```diff
@@ -136,6 +136,10 @@ func connectToConsul(conf *ConfigurationStruct) (*ConfigurationStruct, error) {
 			return conf, errors.New("type check failed")
 		}
 		conf = actual
+		//Check that information was successfully read from Consul
+		if len(conf.Persistence) == 0 {
```

Comment:
This looks like a tricky way to know if configuration have been read or not. Is not possible to perform the check in another way?
If we have to perform this check in order to be sure the configuration it's fine, maybe we should implement similar checks in the other microservices.

Reply:
@jduranf I agree the question now is should we do this in other microservices. Considering that. What's happening is that there's some timing issue between when the support-logging container reads from Consul and when config-seed writes to Consul. In this case, Consul is up but the config-seed hasn't written the necessary key yet, so the service receives a blank structure. Since no error is thrown, I don't know of any other way to test that we got the content we're looking for. In the above scenario, I have to then return a nil up the stack in order to force the initializeConfiguration() function to be called again.